rmw_zenoh_cpp/src/rmw_zenoh.cpp

```diff
@@ -926,10 +911,6 @@ rmw_create_subscription(
     context_impl,
     "unable to get rmw_context_impl_s",
     return nullptr);
-  if (!context_impl->session_is_valid()) {
-    RMW_SET_ERROR_MSG("zenoh session is invalid");
-    return nullptr;
-  }
```

Comment:
Again, why remove this here and in other functions below?

Reply:
As stated above, the `z_check` has been removed.

Reply:
Addressed in https://github.com/ZettaScaleLabs/rmw_zenoh/commit/62ca8225256782d57ac376c826975459aea9b417